### ISSUE TITLE
<http://data.crystalbridges.org> has no data

### ISSUE THREAD

**Issue by workergnome** [american-art/cbm#72]
see <http://data.crystalbridges.org>

It should have a label, at least.

**Comment by bsnikhila** [comment]
For data.crystalbridges.org and data.thewalters.org, the data is shown on the review site, but it will not be shown on Pubby, as Pubby does not support it. All the other owner URLs will be shown on Pubby.

**Comment by kateblanch** [comment]
Why doesn't Pubby support our data?

**Comment by caknoblock** [comment]
It does support your data, just not the top level domain query.  Try this: 
http://data.thewalters.org/object/1727 <http://data.thewalters.org/object/1727>

The problem arises because pubby is only designed to support a single domain, but we needed to have all of the data in a single triple store with multiple domains.

**Comment by bsnikhila** [comment]
Yes, I am very sorry for phrasing it wrong. I meant only the Ownership data for WAM and CBM (the owner label and skos:exactMatch) will not be shown on Pubby. All the objects and constituents will be shown.
